### PR TITLE
I/O homework solution

### PR DESCRIPTION
**Pull request by sohbaker**
- Updates the homework instructions in the README file. 
- Adds a solution as well as tests and test resources. 
- Updates the previously empty example file that students should load and print to the screen in the first part of the homework

### CODE REVIEW

**Review by Genyus — CHANGES_REQUESTED** [review]
Again, there are upstream and solution-only changes combined in a [single commit](https://github.com/cbfacademy-solutions/java-course-exercises/pull/8/commits/3e6108e8dd3179557a36abe228dfb758f93ea2bb). The changes to `example.properties` need to be in a standalone commit so that it can be cherry-picked upstream.

File: homework/input-output/src/main/java/com/cbfacademy/App.java [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        String existingFilePath = "homework/input-output/src/test/resources";
+
+        HashMap<String, String> propMap = new HashMap<String, String>() {
+            {
+                put("db.url", "localhost");
+                put("db.port", "5353");
+                put("db.user", "cbfacademy");
+                put("db.password", "password");
```
Comment (tanyapowell) [inline homework/input-output/src/main/java/com/cbfacademy/App.java:15]: Should the keys and values match:

```
datasource.url=jdbc:postgresql://localhost:port_no/db_name
datasource.username=postgres
datasource.password=password
```

**Review by tanyapowell — CHANGES_REQUESTED** [review]

File: homework/input-output/src/main/java/com/cbfacademy/App.java (hunk 1 above)
Comment (sohbaker) [reply homework/input-output/src/main/java/com/cbfacademy/App.java:15]: Heya, this is separate exercise. There's one step where they need to read from an existing file (`example.properties`) and another step where they need to create a new file with given values (`configuration.properties`)

**Comment by sohbaker** [comment]
Raising a new PR

**Pull request closed without merging**